doc/design/gan_api.md

```diff
@@ -19,12 +19,12 @@ In our GAN design, we wrap it as a user-friendly easily customized python API to
 | softmax loss (done)       | ?                 | Y        |
 | reshape op (done)         | ?                 | Y        |
```

Comment:
I think it is better to create an issue to place this information.
use a `- [ ] zzz` to create checkboxes to follow up the process.